src/legacy/core_plugins/ui_metric/server/routes/api/ui_metric.ts

```diff
@@ -45,36 +56,46 @@ export function registerUiMetricRoute(server: Server) {
     options: {
       validate: {
         payload: Joi.object({
-          report: Joi.object({
-            uiStatsMetrics: Joi.object()
-              .pattern(
-                /.*/,
-                Joi.object({
-                  key: Joi.string().required(),
-                  type: Joi.string().required(),
-                  appName: Joi.string().required(),
-                  eventName: Joi.string().required(),
-                  stats: Joi.object({
-                    min: Joi.number(),
-                    sum: Joi.number(),
-                    max: Joi.number(),
-                    avg: Joi.number(),
-                  }).allow(null),
-                })
-              )
-              .allow(null),
-          }),
+          reportVersion: Joi.number().optional(),
+          userAgent: Joi.object()
+            .pattern(
+              /.*/,
+              Joi.object({
+                key: Joi.string().required(),
+                type: Joi.string().required(),
+                appName: Joi.string().required(),
+                userAgent: Joi.string().required(),
+              })
+            )
+            .allow(null)
+            .optional(),
+          uiStatsMetrics: Joi.object()
+            .pattern(
+              /.*/,
+              Joi.object({
+                key: Joi.string().required(),
+                type: Joi.string().required(),
+                appName: Joi.string().required(),
+                eventName: Joi.string().required(),
+                stats: Joi.object({
+                  min: Joi.number(),
+                  sum: Joi.number(),
+                  max: Joi.number(),
+                  avg: Joi.number(),
+                }).allow(null),
+              })
+            )
+            .allow(null),
         }),
       },
     },
     handler: async (req: any, h: any) => {
-      const { report } = req.payload;
-
       try {
+        const report = req.payload;
         await storeReport(server, report);
-        return {};
+        return { status: 'ok' };
       } catch (error) {
-        return new Boom('Something went wrong', { statusCode: error.status });
+        return h.response({ status: 'fail' }).code(200);
```

Comment:
```suggestion
        return { status: 'fail' };
```

nit: Should be able to do the same as above if expecting a 200 returned.